crates/node/src/interactive_cli/identity.rs

```diff
@@ -80,25 +87,57 @@ impl IdentityCommand {
 
         match self.subcommand {
             IdentitySubcommands::List { context } => {
-                list_identities(node, context, &ind.to_string())?;
+                list_identities(node, Some(context), &ind.to_string())?;
             }
             IdentitySubcommands::New => {
                 create_new_identity(node, &ind.to_string());
             }
             IdentitySubcommands::Alias { command } => {
                 handle_alias_command(node, command, &ind.to_string())?;
             }
+            IdentitySubcommands::Use { identity, context } => {
+                let context_id = node
+                    .ctx_manager
+                    .resolve_alias(context, None)?
+                    .ok_or_eyre("unable to resolve context")?;
+
+                let identity_id = node
+                    .ctx_manager
+                    .lookup_alias(identity, Some(context_id))?
+                    .ok_or_eyre("unable to resolve identity")?;
+
+                let default_alias: Alias<PublicKey> =
+                    "default".parse().expect("'default' is a valid alias name");
+
+                node.ctx_manager
+                    .create_alias(default_alias, Some(context_id), identity_id)?;
+
+                println!(
+                    "{} Default identity set to: {} for context {}",
+                    ind,
+                    identity.cyan(),
+                    context_id.cyan()
+                );
+            }
         }
 
         Ok(())
     }
 }
 
-fn list_identities(node: &Node, context: Alias<ContextId>, ind: &str) -> EyreResult<()> {
-    let context_id = node
-        .ctx_manager
-        .resolve_alias(context, None)?
-        .ok_or_eyre("unable to resolve")?;
+fn list_identities(node: &Node, context: Option<Alias<ContextId>>, ind: &str) -> EyreResult<()> {
```

Comment:
same here

```suggestion
fn list_identities(node: &Node, context: Alias<ContextId>, ind: &str) -> EyreResult<()> {
```

Reply:
factoring https://github.com/calimero-network/core/pull/1171#discussion_r2039935490, we can leave this too